content/views.py

```diff
@@ -153,6 +157,67 @@ def notification(self, request, pk=None, *args, **kwargs):
 
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    @list_route(methods=['get'])
+    def participation(self, request, *args, **kwargs):
+        """Returns true if a user has started a challenge within 2 days of it being created, otherwise false"""
+
+        try:
+            challenge = Challenge.objects.get(state=Challenge.CST_PUBLISHED,
+                                              activation_date__lt=(timezone.now() - timedelta(days=2)))
+        except:
+            return self.response_false
```

Comment:
Storing a response object in the view class isn't idiomatic. The response object is going to carry around weird state.

Create a new response object for each request